FlowerShop/FlowerShop/Scenes/OrderDetails/OrderDetailsViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  OrderDetailsViewController.swift
+//  FlowerShop
+//
+//  Created by Andrei Oltean on 23/02/2020.
+//  Copyright © 2020 Andrei Oltean. All rights reserved.
+//
+
+import UIKit
+
+class OrderDetailsViewController: UIViewController {
+
+    @IBOutlet weak var backButton: UIButton!
+    @IBOutlet weak var flowersImageView: UIImageView!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var addresserLabel: UILabel!
+    @IBOutlet weak var addressLabel: UILabel!
+    
+    var delegate: OrderDetailsCoordinatorDelegate?
+    
+    var orderDetailViewModel: OrderDetailViewModel?
+    let orderDetailsService = OrderDetailsService()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        titleLabel.text = orderDetailViewModel?.description
+        addresserLabel.text = orderDetailViewModel?.addresser
+        addressLabel.text = orderDetailViewModel?.address
+        flowersImageView.image = #imageLiteral(resourceName: "FlowerPlaceholder")
+        
+        if let imageURL = orderDetailViewModel?.imageURL {
+            orderDetailsService.getImage(with: imageURL) { [weak self] result in
+                if case .success(let image) = result {
```

Comment:
I think it's a bit too much logic in the ViewController, calling the service and evaluating the response like this. How can we solve this in another way, but still with MVVM?